src/bindings/template/README.md

```diff
@@ -0,0 +1,39 @@
+- infos =
+- infos/author =
+- infos/licence = BSD
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description =
+
+## Introduction
+
+Explain what this binding does.
+What is its purpose?
+What is its responsibility?
+
+## Files
+
+- Link to the header files.
+- Name of `.so` file.
+
+## Classes
+
+Explain which classes the library contains.
+Add a link to the API docu.
```

Comment:
```suggestion
## Classes/Types

Explain which classes/types the library contains.
Add a link to the API docu.
```

Not every language has classes and some (e.g. Haskell) even use the term for different things

Reply:
9a34f26091794c553e58488a996373e2fb257803